test/Api.Test/AdminConsole/Public/Models/Response/MemberResponseModelTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Bit.Api.AdminConsole.Public.Models.Response;
+using Bit.Core.Entities;
+using Bit.Core.Models.Data;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.AdminConsole.Public.Models.Response;
+
+
+public class MemberResponseModelTests
+{
+    [Fact]
+    public void ResetPasswordEnrolled_ShouldBeTrue_WhenUserHasResetPasswordKey()
```

Comment:
I was following patterns that I’ve seen in the codebase. If anything doesn’t meet standards, please feel free to leave comments, documentation, or suggestions. I’m happy to receive feedback and learn! 😄